tests/Command/CommandTest.php

```diff
@@ -27,54 +28,61 @@ class CommandTest extends WebTestCase
     public function testRunCommandWithoutOptionsAndReuseKernel(): void
     {
         // Run command without options
-        $this->display = $this->runCommand('liipfunctionaltestbundle:test');
-
-        $this->assertInternalType('string', $this->display);
+        $this->commandTester = $this->runCommand('liipfunctionaltestbundle:test');
 
         // Test default values
-        $this->assertContains('Environment: test', $this->display);
-        $this->assertContains('Verbosity level: NORMAL', $this->display);
+        $this->assertContains('Environment: test', $this->commandTester->getDisplay());
+        $this->assertContains('Verbosity level: NORMAL', $this->commandTester->getDisplay());
 
         $this->assertInternalType('boolean', $this->getDecorated());
         $this->assertTrue($this->getDecorated());
 
         // Run command and reuse kernel
-        $this->display = $this->runCommand('liipfunctionaltestbundle:test', [], true);
+        $this->commandTester = $this->runCommand('liipfunctionaltestbundle:test', [], true);
+
+        $this->assertInstanceOf(CommandTester::class, $this->commandTester);
+        $this->assertEquals(0, $this->commandTester->getStatusCode());
 
-        $this->assertContains('Environment: test', $this->display);
-        $this->assertContains('Verbosity level: NORMAL', $this->display);
+        $this->assertContains('Environment: test', $this->commandTester->getDisplay());
+        $this->assertContains('Verbosity level: NORMAL', $this->commandTester->getDisplay());
     }
 
     public function testRunCommandWithoutOptionsAndNotReuseKernel(): void
     {
         // Run command without options
-        $this->display = $this->runCommand('liipfunctionaltestbundle:test');
+        $this->commandTester = $this->runCommand('liipfunctionaltestbundle:test');
+
```

Comment:
We can reduce calls to `getDisplay()` by defining `$this->display` like before:
```suggestion
$this->display =  $this->commandTester->getDisplay();
```

Reply:
@alexislefebvre can we re-have your input on it? 
(About having to use `$this->display` or just a local variable `$display`. Because having only `$display` would change all lines too. )

Reply:
I was thinking about using `$this->display` in the test like before, to reduce the number of changed lines. We can live with this. :+1: